.github/workflows/swift-toolchain.yml

```diff
@@ -2076,8 +2076,7 @@ jobs:
               -p:CERTIFICATE=${env:CERTIFICATE} `
               -p:PASSPHRASE=${{ secrets.PASSPHRASE }} `
               -p:BundleFlavor=offline `
-              -p:VCRedistInstaller="${env:VCToolsRedistDir}\vc_redist.${env:VSCMD_ARG_TGT_ARCH}.exe" `
-              -p:VSVersion=${env:VSCMD_VER} `
+              -p:VCRedistDir="${env:VCToolsRedistDir}\${env:VSCMD_ARG_TGT_ARCH}\Microsoft.VC143.CRT\" `
```

Comment:
Self blocking until https://github.com/apple/swift-installer-scripts/pull/256 is merged